fastestimator/util/data.py

```diff
@@ -60,7 +60,7 @@ def write_without_log(self, key: str, value: Any) -> None:
         """
         self.maps[1][key] = value
 
-    def read_logs(self, extra_keys: Set[str]) -> Dict[str, Any]:
+    def read_logs(self, extra_keys: Set[str] = set()) -> Dict[str, Any]:
```

Comment:
we don't need the extra_keys functionality anymore, but also "= set()" in function declaration is against best practice with mutable default args

Reply:
done